packages/auth/src/access/service.test.ts

```diff
@@ -94,10 +81,7 @@ describe('Access Service', (): void => {
         assetScale: 9
       },
       expiresAt: new Date().toISOString(),
-      description: 'this is a test',
-      externalRef: v4(),
-      receivingAccount: 'test-account',
-      receivingPayment: 'test-payment'
+      receiver: 'test-account'
```

Comment:
This is also a `url`
https://github.com/interledger/open-payments/blob/main/auth-server-open-api-spec.yaml#L628-L630